core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -46,10 +49,12 @@
 		camelVersionToFuseBOMMapping.put("2.15.1.redhat-621117", "6.2.1.redhat-117");
 		camelVersionToFuseBOMMapping.put("2.17.0.redhat-630187", "6.3.0.redhat-187");
 		camelVersionToFuseBOMMapping.put("2.17.3",               "6.3.0.redhat-187");
-		camelVersionToFuseBOMMapping.put("2.18.1",               "6.3.0.redhat-187");
```

Comment:
there is no FUSE BOM version available for 2.18.1 productized version.
2.18.1 productized version is not supported on Fuse, it is only for FIS.